package/yast2-network.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Thu Jun 25 14:15:06 UTC 2015 - mfilka@suse.com
+
+- bnc#922765
```

Comment:
This is the SP4 bug, while yast/yast-yast2#376 mentions [bnc#923990](https://bugzilla.suse.com/show_bug.cgi?id=923990), an openSUSE bug. I think both PRs should mention both bugs.